### PR TITLE
Add meta tag for description to adjust google description

### PR DESCRIPTION
**Pull request by jhack32**
Any suggestions on what the description should be? I just combined the facebook about and some of the text on the website.

### CODE REVIEW

**Review by dijonkitchen — CHANGES_REQUESTED** [review]
Add a period at end, but that's good for now. 

Can you see how it looks in the validators that Facebook, Google, and Twitter have?

**Comment by jhack32** [comment]
It's good now, but for the facebook image, should we use the NYC WTC image instead of our logo?

**Comment by aattsai** [comment]
But WTC is already the cover photo

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <meta property="og:description" content="Cortlandt Labs is a technology consulting agency started in NYC." />
   <meta property="og:site_name" content="Cortlandt Labs"/>
   <meta property="og:url" content="http://cortlandtlabs.co/"/>
-  <meta property="og:image" content="https://raw.githubusercontent.com/cortlandtlabs/cortlandtlabs.github.io/master/assets/Cortlandt-Labs-logo.png"/>
+  <meta property="og:image" content="http://cortlandtlabs.co/assets/Cortlandt-Labs-logo.png"/>
```
Comment (dijonkitchen) [inline index.html:15]: Still think /assets/ works.

**Review by dijonkitchen — APPROVED** [review]

**Comment by dijonkitchen** [comment]
I'm not sure how it looks, but our logo should be good.

**Comment by jhack32** [comment]
Facebook's example shows og:image using a http link, and the other examples I've seen are using HTTP links. I think open graph takes the content exactly as is and if we use a static link, it will just be going to '/assets/cortlandt-labs-logo.png' rather than 'http://cortlandtlabs.co/assets/cortlandt-labs-logo.png'. We should set the width and height though because the image is too big I think

**Pull request merged**